src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,321 @@
+package duke;
+
+import duke.command.EmptyCommand;
+import duke.command.IllegalToDoException;
+import duke.command.InvalidCommandException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    public static final String SAVEFILE_SEPERATOR = "\\|";
+    public static final String FILE_PATH = "duke.txt";
+    public static final String LOGO = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    public static final String LINE = "\t____________________________________________________________\n\t";
+    public static final String BYE = LINE
+            + "Bye. try not to procrastinate!\n"
+            + LINE;
+    public static final String GREETING = LINE
+            + "Hello! I'm Anderson\n\t"
+            + "What do you need to do?\n"
+            + LINE;
+
+    public static String getCommand(String userInput) {
```

Comment:
Why are some methods private and some public? Seems to me that they are all only used in this class